Library/Homebrew/formula_installer.rb

```diff
@@ -212,6 +212,11 @@ def prelude
   def verify_deps_exist
     begin
       compute_dependencies
+    rescue CoreTapFormulaUnavailableError => e
+      raise unless Homebrew::API::Bottle.available? e.name
+
+      Homebrew::API::Bottle.fetch_bottles(e.name)
+      retry
```

Comment:
This looks like it can `retry` forever, can you explain how/why it cannot? A `@retry_count` might be nice to avoid that.

Reply:
Theoretically, I guess it could retry forever, so adding a count might be nice. The idea is that if the formula is available via the API, we'll fetch it so that the next time it runs it won't throw an error because the formula will be found. The way it could retry forever is if the formula is available but, for some reason, the fetch doesn't work (but also doesn't error) in which case it would keep trying to re-fetch.

Reply:
Cool 👍🏻